### ISSUE TITLE
[0.6.8 Windows] Interface does not scale properly with screen resolution

### ISSUE THREAD

**Issue by JimmyCushnie** [teriflix/scrite#247]
Hello,

I'm using Scrite on my 4k monitor and the interface is far too small.

![image](https://user-images.githubusercontent.com/32693298/120977279-29061c00-c741-11eb-929f-1842af6e2fd9.png)

I apologize if there is already a fix for this. I looked around the settings menu and searched this repository but could not find one.

**Comment by pnudupa** [comment]
@JimmyCushnie: I am using a 4k monitor, but on a Mac though. 

It is true that the initial "welcome" scales bad. For now, however, you could perhaps zoom in a bit using the slider on the bottom right.

Here's what I see.

<img width="3840" alt="4k-1" src="https://user-images.githubusercontent.com/8263701/120978641-50d1a000-c792-11eb-8b16-730211a30f91.png">
<img width="3840" alt="4k-2" src="https://user-images.githubusercontent.com/8263701/120978665-59c27180-c792-11eb-8e42-214eeb72c5c4.png">

May I also invite you to take a look at this: https://www.scrite.io/index.php/advanced-editing-features/#chapter16_dpi_setting 

Lets leave this ticket open until we are able to fix the zoom levels in the initial "welcome screen"

**Comment by pnudupa** [comment]
@JimmyCushnie: 

> May I also invite you to take a look at this: https://www.scrite.io/index.php/advanced-editing-features/#chapter16_dpi_setting

Scroll down to the section that reads "DPI Setting" incase the browser doesn't automatically scroll to it.

**Comment by JimmyCushnie** [comment]
Yes, I am able to zoom in on the script contents themselves. My concern is with the interface, for example the menu bars at the top and bottom of the window. For my preference, the interface is much too small, and it is difficult to see and click on. It looks too small for me in your images as well. My interface is much smaller than it appears in Scrite official media, for example the first in-app image on [this page](https://www.scrite.io/index.php/quick-start/).

I am unsure what the DPI setting is supposed to do (DPI of the interface? DPI of printed pages?) but it does not fix my issue.

**Comment by JimmyCushnie** [comment]
To avoid any communication problems, I have drawn red boxes around the parts of the program that are too small for me. I want to make these things about twice as big as they currently are.

![image](https://user-images.githubusercontent.com/32693298/120980769-f1996e80-c744-11eb-8e52-69132c69cceb.png)

**Comment by pnudupa** [comment]
@JimmyCushnie: Ah, I understand now. The Scrite UI doesn't intend to scale that way. For that matter neither does Word, LibreOffice, Davinci Resolve, Chrome, Firefox, Edge or any of the other apps. Even the text on macOS's own menu bar looks small, as you can see.

Microsoft Word at 100% resolution on my 4k monitor:
<img width="3840" alt="Screenshot 2021-06-07 at 1 31 42 PM" src="https://user-images.githubusercontent.com/8263701/120980920-ba52ae00-c794-11eb-9ecd-026ee9ae6b49.png">

Edge on 4K monitor:
![Screenshot 2021-06-07 at 1 36 09 PM](https://user-images.githubusercontent.com/8263701/120981696-8deb6180-c795-11eb-880b-b66afe751859.jpg)

We assume, just like all the other apps, that if you are using 4k display at 4k resolution then it means that you intend for font sizes to "scale down" with the large resolution. 

You can configure the display scaling in Windows Settings to scale all apps.

![image](https://user-images.githubusercontent.com/8263701/120982117-f9cdca00-c795-11eb-926b-4bb084f37a5a.png)

**Comment by JimmyCushnie** [comment]
Yes, I have my display scaling set to 175%, which is comfortable with me for most apps.

![image](https://user-images.githubusercontent.com/32693298/120984173-64581900-c748-11eb-82c0-28efb4ce811c.png)

Edge, for instance, has a very nicely-sized interface, significantly bigger than your example screenshot:

![image](https://user-images.githubusercontent.com/32693298/120982705-f9f2a900-c746-11eb-8c60-383b3989dd74.png)

Is it possible that a bug is preventing Scrite from properly respecting the Windows scale settings? Compared to Edge (and all my other apps) Scrite has a very different interface size.

**Comment by pnudupa** [comment]
When I connect my Windows PC to the 4K display, this is what Windows Explorer looks lilke.

![Screenshot 2021-06-07 at 1 43 43 PM](https://user-images.githubusercontent.com/8263701/120982655-8e382c80-c796-11eb-8944-96a3e21c45d9.jpg)

Ofcourse the text sizes are really really small. Even text on the Settings App looks very small.

![Screenshot 2021-06-07 at 1 45 58 PM](https://user-images.githubusercontent.com/8263701/120982943-dbb49980-c796-11eb-8768-a29023a46acc.jpg)

But when I change the scale to 200%, then the UI looks a lot better.

![Screenshot 2021-06-07 at 1 46 05 PM](https://user-images.githubusercontent.com/8263701/120983032-f2f38700-c796-11eb-9fe6-73135bd927f3.jpg)

As does the Scrite App.

![Screenshot 2021-06-07 at 1 51 36 PM](https://user-images.githubusercontent.com/8263701/120983704-9349ab80-c797-11eb-9b5d-451082440d00.jpg)

Here is the Scrite App at 300% scale.

![Screenshot 2021-06-07 at 1 54 14 PM](https://user-images.githubusercontent.com/8263701/120984240-1965f200-c798-11eb-87f0-131e26f795c3.jpg)

**Comment by JimmyCushnie** [comment]
Okay, I have now tested thoroughly and I can conclude that Scrite's interface does not change size with my display scaling setting.

Here is the program at 100% scaling:

![image](https://user-images.githubusercontent.com/32693298/120984924-1c85c180-c749-11eb-98bc-deaf0d06d861.png)

Here is the program at 300% scaling (after restarting the program):

![image](https://user-images.githubusercontent.com/32693298/120984812-fd872f80-c748-11eb-86c5-33b4c0d1cec0.png)

Very odd that it works properly for you, but not for me.

Thanks very much for your help so far. Should I close this issue and make a new one for "scrite does not respect my display scale setting?"

**Comment by pnudupa** [comment]
@JimmyCushnie: Actually it scales proportionately on my Windows PC, as you can see from the screenshots attached. Not sure what's going on with the app on your PC. 

Have you tried restarting Scrite after changing the scale settings in Windows?

Let's keep this ticket open until we at least know what is going on.

**Comment by JimmyCushnie** [comment]
Yes, the interface does not change size even after multiple restarts of the program.

The window title bar *does* change size, however, as you can see in my screenshots.

**Comment by pnudupa** [comment]
@JimmyCushnie: Okay, thanks for confirming. I can see that the UI doesn't scale on your PC. Let's keep this ticket open for now.

**Comment by JimmyCushnie** [comment]
Hi again. Off topic, but I've just discovered the Scrite forums. Should I use the forums or github issues to submit future bugs and feature suggestions? It is not clear to me which method of communication is preferred by the developers.

**Comment by pnudupa** [comment]
@JimmyCushnie: So far the consensus has been to start a thread on the Forum and then to file tickets here as required. It would be great if we can begin conversations on the Forum first. 😀

**Comment by JimmyCushnie** [comment]
Sounds good, I will do that from now on! I think you should add that information to the readme, so other folks who discover Scrite through github like I did can be aware of it.

**Comment by pnudupa** [comment]
Yes, good idea. Will add a note in README.
